### PR TITLE
SC-585 API callable complexities

### PR DESCRIPTION
**Pull request by xrtm000**

### CODE REVIEW

File: node/src/test/scala/com/wavesplatform/http/UtilsRouteSpec.scala [hunk 1]
```diff
@@ -174,6 +184,31 @@ class UtilsRouteSpec extends RouteSpec("/utils") with RestAPISettingsHelper with
...
 
+  val dApp =
```
Comment (msmolyakov) [inline node/src/test/scala/com/wavesplatform/http/UtilsRouteSpec.scala:187]: please add Verifier function to this script

Comment (xrtm000) [reply node/src/test/scala/com/wavesplatform/http/UtilsRouteSpec.scala:187]: +

File: node/src/test/scala/com/wavesplatform/http/UtilsRouteSpec.scala [hunk 2]
```diff
@@ -174,6 +184,34 @@ class UtilsRouteSpec extends RouteSpec("/utils") with RestAPISettingsHelper with
...
+      |
+      |@Callable(i)
+      |func writeAndSendWaves(value: Int, recipient: String, amount: Int) = {
+      |    ScriptResult(
+      |        WriteSet([DataEntry("result", value)]),
+      |        TransferSet([ScriptTransfer(Address(recipient.fromBase58String()), amount, unit)])
+      |    )
+      |}
+      |
+      |@Verifier(tx)
```
Comment (msmolyakov) [inline node/src/test/scala/com/wavesplatform/http/UtilsRouteSpec.scala:211]: also check explicitly that without the verifier function the `complexity` field is 0

**Comment by msmolyakov** [comment]
what if DApp script is empty (contains only directives)?

File: node/src/test/scala/com/wavesplatform/http/UtilsRouteSpec.scala (hunk 2 above)
Comment (xrtm000) [reply node/src/test/scala/com/wavesplatform/http/UtilsRouteSpec.scala:211]: Ok

**Pull request merged**